src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discount3 = 20;
+  const discount7 = 50;
+  const finalPrice = price * days;
+
+  if (days >= 7) {
+    return finalPrice - discount7;
+  }
+
+  if (days >= 3) {
+    return finalPrice - discount3;
+  } else {
+    return finalPrice;
+  }
```

Comment:
```suggestion
  if (days >= 3) {
    return finalPrice - discount3;
  }
  
  return finalPrice;
```
No need for else here. You code will not go further after `return` anyway